steps/run-tests-using-marathon-cloud/0.1.0/step.yml

```diff
@@ -0,0 +1,61 @@
+title: Run tests using Marathon Cloud
+summary: |
+  Run all your automated tests in just 15 minutes, no matter how many tests you have
+description: |
+  With this action you can easily run your Android instrumentation or iOS tests. Marathon Cloud is designed to run all your automated tests in just 15 minutes, no matter how many tests you have.
+website: https://marathonlabs.io
+source_code_url: https://github.com/MarathonLabs/bitrise-step-run-tests-using-marathon-cloud
+support_url: https://github.com/MarathonLabs/bitrise-step-run-tests-using-marathon-cloud/issues
+published_at: 2023-06-27T12:45:44+04:00
+source:
+  git: https://github.com/MarathonLabs/bitrise-step-run-tests-using-marathon-cloud.git
+  commit: ed0dd7a013f0b4bd3f51e0e4bfe64148c344b96b
+project_type_tags:
+  - android
+  - ios
+type_tags:
+  - test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: malinskiy/tap/marathon-cloud
+inputs:
+  - application: ""
+    opts:
+      title: App File
+      summary: Application binary path, e.g. apk file for Android or zip file of iOS Application
+      description: |
+        **Android**: `application` should point to the APK file. **iOS**: `application` should point to an ARM compatible Simulator build packaged in an ipa format or a zip archive.
```

Comment:
When using the Workflow Editor's Workflows tab for configuring the step, the step input key is not visible, so mentioning it in the description or summary (which fields are presented on the Workflows tab) can be confusing.

![Screenshot 2023-07-04 at 10 05 28](https://github.com/bitrise-io/bitrise-steplib/assets/12427060/f0c24e6b-6461-4be5-a237-264eb2e20ba9)

Also when building for the simulator the generated artifact is a `.app` file and not a `.ipa` file.

I suggest something like this for as a summary and description for this input:

```
      summary: Application binary path.
      description: |-
        Application binary path.  
        For an Android project: app APK file path.  
        For an iOS project: an ARM compatible Simulator build packaged in an APP format or a zip archive.
```

(`summary` and `description` can be in a Markdown format. There are two spaces at the end of the first two lines in the description for presenting them in new lines.)

You can verify the step.yml locally by running: `bitrise :workflow-editor`, if you have a workflow that uses the current [step from a local directory path](https://devcenter.bitrise.io/en/references/steps-reference/step-reference-id-format.html#the-path---source):

```
  verify_in_workflow_editor:
    steps:
    - path::./: {}
```